CHANGELOG

```diff
@@ -5,10 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] 2024-03-04
+
+- This version represents the first public release of the research catalog refactor.
```

Comment:
Nit: capitalize Research Catalog.

Reply:
Ended up removing this line and creating a Release Notes header

Reply:
RC should still be capitalized.